src/applications/personalization/common/mocks/feature-toggles/index.js

```diff
@@ -1,64 +1,59 @@
-const generateFeatureToggles = (toggles = {}) => {
-  const {
-    authExpVbaDowntimeMessage = true,
-    myVaEnableNotificationComponent = true,
-    myVaHideNotificationsSection = true,
-    myVaUseExperimental = true,
-    myVaUseExperimentalFrontend = true,
-    myVaUseExperimentalFullstack = true,
-    myVaNotificationDotIndicator = true,
-    myVaEnableMhvLink = true,
-    myVaUpdateErrorsWarnings = true,
-    vaOnlineSchedulingStaticLandingPage = true,
-  } = toggles;
+const { keyBy } = require('lodash');
+const FEATURE_FLAG_NAMES = require('../../../../../platform/utilities/feature-toggles/featureFlagNames.cjs');
```

Comment:
this module is passed directly to mocker-api, running in CommonJS mode, so i don't believe there's a way to use platform aliases in babel config, unless we were to pre-compile the 3rd party code when running the local mock server (which seems excessive imo). using cjs module seems a bit simpler, since it's only supporting the local dev use-case.

is there another way to get this relative path, w/o so much backtracking? since this is a cjs module, and only needed in node, we could use file and path modules here, but if there's a prefered way of doing this in common js modules for va.gov, let me know :)

Reply:
You could do:

``` js
const path = require('path');

const featureTogglePath = path.join(
  process.cwd(),
  '/src/platform/utilities/feature-toggles/featureFlagNames.cjs',
);

const FEATURE_FLAG_NAMES = require(featureTogglePath);
```

But eslint complains with this error: 'Calls to require() should use string literals eslint [import/no-dynamic-require](https://github.com/import-js/eslint-plugin-import/blob/v2.24.2/docs/rules/no-dynamic-require.md)' So i'd say leave it as it is.